cmake/public/LoadHIP.cmake

```diff
@@ -150,6 +150,10 @@ IF(HIP_FOUND)
   #find_package_and_print_version(hipsparse REQUIRED)
   find_package_and_print_version(rocsparse REQUIRED)
 
+  find_package(rocprim REQUIRED CONFIG PATHS "/opt/rocm/rocprim")
```

Comment:
Better to follow the convention of other rocm libraries (i.e. define `ROCPRIM_PATH`, `rocprim_DIR` and call `find_package_and_print_version`